recipes/libssh/all/conanfile.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout
+from conan.tools.files import collect_libs, copy, get, rmdir
+
+
+class LibSSHRecipe(ConanFile):
+    name = "libssh"
+    version = "0.10.4"
```

Comment:
```suggestion
```
please, do not add version in recipes, so we can re-use the same recipe for any version.